tpot/base.py

```diff
@@ -1018,10 +1046,16 @@ def _evaluate_individuals(self, individuals, features, target, sample_weight=Non
 
     @_pre_test
     def _mate_operator(self, ind1, ind2):
-        return cxOnePoint(ind1, ind2)
+        for __ in range(self.MAX_MUT_LOOPS):
```

Comment:
Hm yeah, I used two so it did not have the same name as the second return argument from the crossover function, because that value is also not used. I wasn't quite sure what to do here, I played around a bit to see what pylint recommended, but nothing came of that either.
I'll change it back to one.